### ISSUE TITLE
Can become a group organiser

### ISSUE THREAD

**Issue by kevbite** [YorkDevelopers/venimus#2]
**As a** Group Organiser
**I want** to be able to create a unique group
**So that** I can arrange meetups

**Comment by DavidBetteridge** [comment]
We "could" go live with just our meetings without this being in place.

**Comment by kevbite** [comment]
This is true, as long as we're not constantly hacking the database to provide functionality manually.
